integration_tests/sdk/requirement_test.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
+import pytest
+from aqueduct.error import AqueductError
+
+from constants import SENTIMENT_SQL_QUERY
+from utils import get_integration_name
+from aqueduct import op
+
+INVALID_REQUIREMENTS_PATH = "~/random.txt"
+VALID_REQUIREMENTS_PATH = "requirements/requirements.txt"
+
+
+def check_if_transformers_exist():
+    try:
+        import transformers
+    except ImportError:
+        return False
+    return True
+
+
+@op(reqs_path=INVALID_REQUIREMENTS_PATH)
+def invalid_valid_path_operator(table: pd.DataFrame) -> pd.DataFrame:
+    return table
+
+
+@op(reqs_path=VALID_REQUIREMENTS_PATH)
+def valid_sentiment_prediction(reviews: pd.DataFrame) -> pd.DataFrame:
+    import transformers
+
+    model = transformers.pipeline("sentiment-analysis")
+    return reviews.join(pd.DataFrame(model(list(reviews["review"]))))
+
+
+@op()
+def default_sentiment_prediction(reviews: pd.DataFrame) -> pd.DataFrame:
```

Comment:
can we name these clearer (its not obvious that its the reqs path that is invalid/valid):
```
sentiment_prediction_without_reqs_path()
```
and above would be:
```
sentiment_prediction_with_valid_reqs_path()
```
and `sentiment_prediction_with_invalid_reqs_path()`